scripts/pull_stats.py

```diff
@@ -18,37 +19,41 @@
 urllib3.disable_warnings()
 
 if __name__ == "__main__":
+    for game in ['uya', 'dl']:
+        protocol: str = CREDENTIALS[game]["horizon_middleware_protocol"]
+        host: str = CREDENTIALS[game]["horizon_middleware_host"]
+        horizon_app_id: int = CREDENTIALS[game]["horizon_app_id"]
+        horizon_username: str = CREDENTIALS[game]["horizon_middleware_username"]
+        horizon_password: str = CREDENTIALS[game]["horizon_middleware_password"]
+
+        token: str = authenticate(
+            protocol=protocol,
+            host=host,
+            username=horizon_username,
+            password=horizon_password
+        )
 
-    protocol: str = CREDENTIALS["horizon_middleware_protocol"]
-    host: str = CREDENTIALS["horizon_middleware_host"]
-    horizon_app_id: int = CREDENTIALS["horizon_app_id"]
-    horizon_username: str = CREDENTIALS["horizon_middleware_username"]
-    horizon_password: str = CREDENTIALS["horizon_middleware_password"]
-
-    token: str = authenticate(
-        protocol=protocol,
-        host=host,
-        username=horizon_username,
-        password=horizon_password
-    )
-
-    all_players = get_all_accounts(
-        protocol=protocol,
-        host=host,
-        app_id=horizon_app_id,
-        token=token
-    )
-
-    all_stats: dict = dict()
-
-    for leaderboard_item in tqdm(all_players, desc="Pulling stats from Horizon Production..."):
-        all_stats[leaderboard_item["AccountId"]] = get_account_basic_stats(
+        all_players = get_all_accounts(
             protocol=protocol,
             host=host,
-            account_id=leaderboard_item["AccountId"],
             app_id=horizon_app_id,
             token=token
         )
 
-    with open("stats.json", "w") as stream:
-        json.dump(all_stats, stream)
+        all_stats: dict = dict()
+
+        start_time = datetime.now()
```

Comment:
This should have a type definition. I think this one is important because times can be tracked in many formats.